airflow/providers/google/marketing_platform/example_dags/example_display_video.py

```diff
@@ -94,4 +105,30 @@
     )
     # [END howto_google_display_video_deletequery_report_operator]
 
+    # [START howto_google_display_video_sdf_to_gcs_operator]
+    download_sdf_file = GoogleDisplayVideo360SDFtoGCSOperator(
+        task_id="download_sdf_file",
+        body_request=BODY_REQUEST,
+        bucket_name=BUCKET,
+        object_name=OBJECT_NAME,
+        gzip=False,
+    )
+    # [END howto_google_display_video_sdf_to_gcs_operator]
+
+    # [START howto_google_display_video_gcs_to_big_query_operator]
+    upload_sdf_to_bigquery = GCSToBigQueryOperator(
+        task_id='upload_to_bigquery',
+        bucket=BUCKET,
+        source_objects=['{{ task_instance.xcom_pull("task_sdf_to_gcs_id")}}'],
+        destination_project_dataset_table='airflow_test.gcs_to_bq_table',
```

Comment:
Should additional step be added in system test to create this dataset? Or will it be create by this operator?

Reply:
As far as I know dataset is created by operator and also deleted.

Reply:
True, the table is created but it's not removed. Just in case I would add create dataset and remove dataset in test.